bosi/rhosp_resources/build_scripts/rhosp_packager.sh

```diff
@@ -2,16 +2,45 @@
 
 # Following build params expected for this script:
 # OpenStackBranch
-# BosiBranch
-# RHOSPVersion
+# BcfBranch
 # Revision
+# IvsBranch (optional)
+
+# mapping for OpenStackBranch to RHOSPVersion, default is latest = 9
+# occasionally cleanup when we stop supporting certain versions
+RHOSPVersion="9"
+case "$OpenStackBranch" in
+  *"mitaka"*) RHOSPVersion="9" ;;
+  *"liberty"*) RHOSPVersion="8" ;;
+  *"kilo"*) RHOSPVersion="7" ;;
+esac
+
+# if IvsBranch is not specified, it is same as BcfBranch
+if [ -z  $IvsBranch  ]
+then
+    IvsBranch="$BcfBranch"
+fi
+
+# if branch is not master, append 'v' e.g. v3.7.0
+if [ $IvsBranch != "master" ]
+then
+    IvsBranch="v$IvsBranch"
+fi
+
+# if BcfBranch is not master, append 'bcf-' to it
+BosiBranch="$BcfBranch"
```

Comment:
i ran it with the test job here: https://jenkins.bigswitch.com/job/test_rhosp_job/3/console
i'll have to update the main job once the PR is ready for merge